app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -87,6 +91,12 @@ class ProfileChooserFragmentPresenter @Inject constructor(
     return viewModelProvider.getForFragment(fragment, ProfileChooserViewModel::class.java)
   }
 
+  private fun initAdapter() {
+    val layoutManager = GridAutoFitLayoutManager(activity.applicationContext, 500)
+    binding.profileRecyclerView.setLayoutManager(layoutManager)
```

Comment:
Convert this to single line 

`binding.profileRecyclerView.layoutManager = GridAutoFitLayoutManager(activity.applicationContext, 500)`

Reply:
Also, what is this 500? Maximum number of profiles visible in a row? If yes, then should we decrease that number?

Reply:
added doc in class its in assumed cell Width in px.

Reply:
I think the first comment in this thread was missed. Please update the code as mentioned below:

```
// Assume cell width of 500px
binding.profileRecyclerView.layoutManager = GridAutoFitLayoutManager(activity.applicationContext, columnWidth = 500)
binding.profileRecyclerView.setHasFixedSize(true)
```

Reply:
Done.